### PR TITLE
multiple changes

### PR DESCRIPTION
**Pull request by garbas**
this PR should be merged once ploneintranet/ploneintranet.prototype#158 and Patternslib/Patterns#413 are merged.

[Review on Reviewable](https://reviewable.io/reviews/patternslib/pat-raptor/1)

### CODE REVIEW

File: src/pat-raptor-link-tooltip.js [hunk 1]
```diff
@@ -27,8 +60,8 @@ define([
   LinkTooltip.prototype = Object.create(Raptor.RaptorPlugin.prototype);
 
   LinkTooltip.prototype.init = function() {
-    this.raptor.bind('change', initializeToolkit)
-    this.raptor.bind('layoutShow', initializeToolkit)
+    this.raptor.bind('change', initializeToolkit(this, 'change'))
+    this.raptor.bind('layoutShow', initializeToolkit(this, 'layoutShow'))
   };
```
Comment (jcbrand) [inline src/pat-raptor-link-tooltip.js:65]: A more canonical JS way would be to use `initializeToolkit.bind(this)`. Then you don't need the inner function inside `initializeToolkit`.

**Comment by jcbrand** [comment]
@garbas: When you make pull requests, please provide a summary of the changes you implemented and the problem you're trying to solve with the pull request.

**Comment by garbas** [comment]
@jcbrand description is in commit messages

File: src/pat-raptor-link-tooltip.js (hunk 1 above)
Comment (garbas) [inline src/pat-raptor-link-tooltip.js:65]: will fix this. i was iteration few times. tnx!

**Comment by garbas** [comment]
@jcbrand fixed things from your comments.

**Pull request merged**